<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-8TP6OQIJ" failures="1" tests="2" name="Test" time="13.514" errors="0" timestamp="2023-04-27T08:24:48 IST">
  <testcase classname="TestCases.VerifyLogin" name="invalidLogin" time="0.567">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error message is not valid expected [Please check your username. If you still can&amp;apos;t log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Error message is not valid expected [Please check your username. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at TestCases.VerifyLogin.invalidLogin(VerifyLogin.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin -->
  <testcase classname="TestCases.VerifySignup" name="signup" time="5.753"/>
</testsuite> <!-- Test -->
